lib/vagrant-service-manager/command.rb

```diff
@@ -6,6 +6,15 @@ module ServiceManager
     DOCKER_PATH = '/home/vagrant/.docker'
     SUPPORTED_SERVICES = ['docker', 'openshift', 'kubernetes']
     SCCLI_SERVICES = ['openshift', 'k8s']
+    KUBE_SERVICES = [
+      "etcd",
+      "kube-apiserver",
+      "kube-controller-manager",
+      "kube-scheduler",
+      "kubelet",
+      "kube-proxy",
+      "docker"
```

Comment:
The above discussion seems like a great reason for having sccli be the "source of all information" about services in the box.